napari/_qt/qt_resources/styles/02_custom.qss

```diff
@@ -302,12 +303,12 @@ QtColorBox {
 /* ----------------- QtLayerControls -------------------- */
 
 QtLayerControls > QLabel, QtLayerControls, QtPlaneControls > QLabeledSlider > QAbstractSpinBox {
-  font-size: 11pt;
+  font-size: {{ font_size }};
```

Comment:
Seems like this should be `font-size: {{ decrease(font_size, 1) }};` (if using 12pt as base font size value)